vs060_moveit_config/.setup_assistant

```diff
@@ -0,0 +1,8 @@
+moveit_setup_assistant_config:
+  URDF:
+    package: vs060
+    relative_path: model/vs060A1_AV6_NNN_NNN.urdf
+  SRDF:
+    relative_path: config/vs060A1_AV6_NNN_NNN.srdf
+  CONFIG:
+    generated_timestamp: 1408435901
```

Comment:
timestamp is kind of dummy -- it's taken from [another existing (and working) package](https://github.com/ros-industrial-consortium/fermi/blob/650df655e52a59531f77e63150ea2c9b2b53e654/fermi_sia20d_moveit_config/.setup_assistant#L8) that I picked at random.